lib/hako/schedulers/ecs.rb

```diff
@@ -1113,6 +1113,11 @@ def print_definition_in_cli_format(definition, additional_env: {})
           if definition[:init_process_enabled]
             cmd << '--init'
           end
+
+          definition[:linux_parameters][:tmpfs].each do |tmpfs|
```

Comment:
`definition[:linux_parameters][:tmpfs]` could be nil

Reply:
0c77e64 ✨